src/lib/select/core/base-select-adapter.ts

```diff
@@ -225,14 +228,13 @@ export abstract class BaseSelectAdapter extends BaseAdapter<IBaseSelectComponent
 
   private _createOptionGroupElement(group: ISelectOptionGroup): HTMLElement {
     const optionGroupElement = document.createElement('forge-option-group');
-    assignMatchingProperties(group, optionGroupElement);
-    optionGroupElement.label = group.text ?? '';
+    Object.assign(optionGroupElement, group);
```

Comment:
We will now just blindly copy everything from the config objects for both options and option groups. The previous strategy of only applying "matching properties" was short sighted.